packages/Badge/src/Badge.tsx

```diff
@@ -5,6 +5,16 @@ import { ccClassName, emotion } from '@leafygreen-ui/lib';
 
 const { css } = emotion;
 
+export type Variant =
```

Comment:
What if this was a string enum instead?

Reply:
I don't really have a preference between unions of strings and enums. This component was already using strings for variants rather than an object, so I went with a union, but if we want to standardize on one pattern I can change it

Reply:
Talked with @DesignerDave in Slack, sounds like having the enum in the emitted js file is enough of a selling point to pick string enums